ui/shadow_priest/presets.ts

```diff
@@ -0,0 +1,229 @@
+			ItemSpec.create({
+				id: 21870, // Frozen Shadoweave Boots
+				enchant: Enchants.BOARS_SPEED,
+				gems: [
+					Gems.RUNED_LIVING_RUBY,
+					Gems.RUNED_LIVING_RUBY,
+				],
+			}),
+			ItemSpec.create({
+				id: 29352, // Cobalt Band of Tyrigosa
+				enchant: Enchants.RING_SPELLPOWER,
+			}),
+			ItemSpec.create({
+				id: 28793, // Band of Crimson Fury
+				enchant: Enchants.RING_SPELLPOWER,
+			}),
+			ItemSpec.create({
+				id: 28789, // Eye of Magtheridon
+			}),
+			ItemSpec.create({
+				id: 29370, // Icon of the Silver Crescent
+			}),
+			ItemSpec.create({
+				id: 29350, // The Black Stalk
+			}),
+			ItemSpec.create({
+				id: 28770, // Nathrezim Mindblade
+				enchant: Enchants.SOULFROST,
+			}),
+			ItemSpec.create({
+				id: 29272, // Orb of the Soul-Eater
+			}),
+		],
+	}),
+};
+
+export const P2_BIS = {
+	name: 'P2 BIS',
+	tooltip: Tooltips.BASIC_BIS_DISCLAIMER,
+	gear: EquipmentSpec.create({
+		items: [
+			ItemSpec.create({
+				id: 32494, // Destruction Holo-gogs
+				enchant: Enchants.GLYPH_OF_POWER,
+				gems: [
+					Gems.MYSTICAL_SKYFIRE_DIAMOND,
+					Gems.GLOWING_NIGHTSEYE,
+				],
+			}),
+			ItemSpec.create({
+				id: 30666// ritssyns-lost-pendant
+			}),
+			ItemSpec.create({
+				id: 30163, // wings-of-the-avatar
+				enchant: Enchants.GREATER_INSCRIPTION_OF_DISCIPLINE,
+				gems: [
+					Gems.RUNED_LIVING_RUBY,
+					Gems.RUNED_LIVING_RUBY,
+				],
+			}),
+			ItemSpec.create({
+				id: -18, // illidari-cloak
+				enchant: Enchants.SUBTLETY,
+			}),
+			ItemSpec.create({
+				id: 30107, // vestments-of-the-sea-witch
+				enchant: Enchants.CHEST_EXCEPTIONAL_STATS,
+				gems: [
+					Gems.RUNED_LIVING_RUBY,
+					Gems.RUNED_LIVING_RUBY,
+					Gems.RUNED_LIVING_RUBY,
+				],
+			}),
+			ItemSpec.create({
+				id: -14, // ravagers-cuffs shadow wrath
+				enchant: Enchants.WRIST_SPELLPOWER,
+			}),
+			ItemSpec.create({
+				id: 28780, // nethersteel-lined-handwraps
+				enchant: Enchants.GLOVES_SPELLPOWER,
+				gems: [
+					Gems.RUNED_LIVING_RUBY,
+					Gems.RUNED_LIVING_RUBY,
+				],
+			}),
+			ItemSpec.create({
+				id: 30038, // Belt of Blasting
+				gems: [
+					Gems.RUNED_LIVING_RUBY,
+					Gems.RUNED_LIVING_RUBY,
+				],
+			}),
+			ItemSpec.create({
+				id: 29972, // trousers-of-the-astromancer
+				enchant: Enchants.RUNIC_SPELLTHREAD,
+				gems: [
+					Gems.RUNED_LIVING_RUBY,
+					Gems.RUNED_LIVING_RUBY,
+					Gems.RUNED_LIVING_RUBY,
+				],
+			}),
+			ItemSpec.create({
+				id: -5, // gliders-foot-wraps
+				enchant: Enchants.BOARS_SPEED,
+			}),
+			ItemSpec.create({
+				id: 30109, // ring-of-endless-coils
+				enchant: Enchants.RING_SPELLPOWER,
+			}),
+			ItemSpec.create({
+				id: 29922, // band-of-alar
+				enchant: Enchants.RING_SPELLPOWER,
+			}),
+			ItemSpec.create({
+				id: 29370, // Icon of the Silver Crescent
+			}),
+			ItemSpec.create({
+				id: 38290, // dark-iron-smoking-pipe
+			}),
+			ItemSpec.create({
+				id: 29982, // wand of forgotten star
+			}),
+			ItemSpec.create({
```

Comment:
I'm not sure we want to be including same-season pvp gear in the bis lists. It's the same thing as world-boss gear: this isn't accessible to the majority of players, and the ones who can get it will have no trouble using the sim to figure that out on their own.

Reply:
I was just letting the spriest people decide for themselves. We can change the bis list now that the sim is functional